wordpress-mobile/WordPress-Android
# [Blogging Prompts] Create Prompts List item

Part of #17125

Add the UI component for the Prompts List item matching UI specs, and use it in the Prompts List screen.

_Note: this is still using mocked data!_

Video Demo:

https://user-images.githubusercontent.com/5091503/210013429-48318b86-b773-4b95-94b1-12358fe4f41d.mp4

To test:
1. Do the steps to enable the Prompts List feature flag found in the test section of #17675
2. Sign in (if not signed in already)
3. Go to the home dashboard (My Site -> Home)
4. **Verify** the Prompts card is shown
5. Tap the 3-dot menu in the Prompts card
6. Tap the "View more prompts" option
7. **Verify** the Prompts List activity is shown
8. **Verify** it shows a "Loading..." text for a while
9. **Verify** the content is updated with a fake list of 11 items, matching the design spec

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
UI changes only, which are not currently covered by tests.

PR submission checklist:
